crates/wash-lib/build.rs

```diff
@@ -25,6 +25,15 @@ async fn record_reachability(host: &str) {
 
 #[tokio::main]
 async fn main() {
+    // Determine whether docker sock is present locally
+    println!("cargo:rustc-check-cfg=cfg(var_run_docker_sock_is_present)");
+    if tokio::fs::metadata("/var/run/docker.sock")
```

Comment:
Please consider https://github.com/testcontainers/testcontainers-rs/pull/721

Reply:
Thanks for pointing this out! I'll use a different check so we don't get bit by that change of defaults whenever they do it.